experimental/entity/entity.go

```diff
@@ -0,0 +1,56 @@
+package entity
+
+// This is the base class for (non-raw) kinds
+type Envelope struct {
+	// UID may contain / to indicate parent folder's
+	UID string `json:"UID,omitempty"`
+
+	// dash, ds, alert, folder, svg, png, df, dqr, ... (will validate body)
+	Kind string `json:"kind,omitempty"`
+
+	// Version of the included body (semver)
```

Comment:
Version of the included body - if it is coming from a coremodel, as the above "Kind" types given in comment ought to - would be a syntactic version, not semver

Reply:
👍